common/netmap.go

```diff
@@ -12,6 +14,11 @@ import (
 // a successive call is not guaranteed.
 // Caller must have `NewEpoch` method with a single numeric argument.
 func SubscribeForNewEpoch() {
+	callingHash := runtime.GetExecutingScriptHash()
```

Comment:
It's executing script hash and you're calling it `calling`.

Reply:
i meant that is a contract that called that method. renamed to `callingContract`, it is ok?